app/src/main/java/org/sil/gatherwords/MainActivity.java

```diff
@@ -31,6 +31,8 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
 
     public void onCreateSessionClick(View v) {
         Intent intent = new Intent(this, SessionActivity.class);
+        // To distinguish between creating a session and viewing the settings of an old one
+        intent.putExtra("creating_session", true);
```

Comment:
Please use a `static final` variable instead of raw string; this allows compiler assisted checks.